File: classes/parte-11-proyecto-integrador-poliglota/167-componente-cli-lenguaje-de-sistemas/implementaciones/c/main.c
#include <stdio.h>

int main(void) {
    char comando[64], t[64];
    if (scanf("%63s", comando) != 1) return 1;
    int args = 0;
    while (scanf("%63s", t) == 1) args++;
    printf("comando=%s args=%d\n", comando, args);
    return 0;
}

File: classes/parte-11-proyecto-integrador-poliglota/167-componente-cli-lenguaje-de-sistemas/implementaciones/java/Main.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String[] t = br.readLine().trim().split("\\s+");
        System.out.println("comando=" + t[0] + " args=" + (t.length - 1));
    }
}
